client/components/google-my-business/stats-nudge.js

```diff
@@ -13,12 +13,14 @@ import Button from 'components/button';
 import DismissibleCard from 'blocks/dismissible-card';
 import SectionHeader from 'components/section-header';
 
+const TWO_WEEKS_IN_SECONDS = 60 * 60 * 24 * 14;
+
 const GMBStatsNudge = ( { translate } ) => {
 	return (
 		<DismissibleCard
 			className="google-my-business__stats-nudge"
 			preferenceName="show-google-my-business-nudge"
```

Comment:
I know it's not related to this pull request's changes, but we usually don't start the preference name with a verb. As such we could remove this `show-` prefix (as otherwise the full preference name would then be `dismissible-card-show-google-my-business-nudge`).